docs/Additional-Activation-OTP.md

```diff
@@ -0,0 +1,168 @@
+# Additional activation OTP
+
+The purpose of additional activation OTP is to help with the user authentication, or with the activation confirmation. The additional OTP can be used either in the early stages of the activation or later when the activation is created and waits for the confirmation in the OTP_USED<sup>1</sup> state.
+
+We will describe each situation in detail in the separate chapters:
+
+1. [Additional user authentication](#additional-user-authentication)
+   - [Regular activation with OTP](#regular-activation-with-otp)
+   - [Custom activation with OTP](#custom-activation-with-otp)
+2. [Activation confirmation](#activation-confirmation)
+   - [Confirm regular activation with OTP](#confirm-regular-activation-with-otp)
+   - [Confirm custom activation with OTP](#confirm-custom-activation-with-otp)
+   - [Confirm activation recovery with OTP](#confirm-activation-recovery-with-otp)
+
+
+> Notes:
+> 1. Be aware that additional activation OTP is not related to OTP_USED state. The naming of OTP_USED state was introduced in PowerAuth Protocol V2, so we keep it due to the technical and historical reasons. The actual meaning of OTP_USED state is that the activation is awaiting the commit.
+>
+> 2. This part of the documentation describes in detail how usage of additional activation OTP changes the activation process. So, before you start, you should be familiar with actors and processes defined for the [regular activation](Activation.md).
+
+## Additional user authentication
+
+In this common scenario, it's expected that the PowerAuth activation is not yet created so that the additional activation OTP can be used in the time of the activation creation as additional user authentication.
+
+### Regular activation with OTP
+
+1. User is authenticated in Master Front-End Application and initiates the activation creation process:
+
+   1. Master Front-End Application generates random activation OTP.
+   1. Master Front-End Application then asks PowerAuth server to create an activation, with using this OTP ([`initActivation`](https://github.com/wultra/powerauth-server/blob/develop/docs/SOAP-Service-Methods.md#method-initactivation) method, OTP validation set to ON_KEYS_EXCHANGE).
+   1. Master Front-End Application then displays QR code, containing an activation code.
+   1. At the same time, Master Front-End Application initiates the delivery of activation OTP. It's recommended to deliver such code via the dedicated out-of-band channel, for example, via SMS.
```

Comment:
`via the dedicated out-of-band channel` -> `via a dedicated out-of-band channel`, we do not know yet what the exact channel is, `the` cannot be used yet